pkg/datapath/fake/datapath.go

```diff
@@ -10,90 +10,103 @@ import (
 	"github.com/cilium/cilium/pkg/datapath"
 	"github.com/cilium/cilium/pkg/datapath/loader/metrics"
 	"github.com/cilium/cilium/pkg/datapath/types"
+	"github.com/cilium/cilium/pkg/testutils/mockmaps"
 )
 
-type fakeDatapath struct {
+var _ datapath.Datapath = (*FakeDatapath)(nil)
```

Comment:
What does this line do?

Reply:
I would say it's meant to ensure that the type `FakeDatapath` implements the interface.

Reply:
Yep, exactly. Asserts that `FakeDatapath` implements `datapath.Datapath`. https://go.dev/doc/effective_go#blank_implements.